Cargo.lock

```diff
@@ -9550,7 +9550,7 @@ version = "0.13.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d0f3e5beed80eb580c68e2c600937ac2c4eedabdfd5ef1e5b7ea4f3fba84497b"
 dependencies = [
- "heck 0.5.0",
+ "heck 0.4.1",
```

Comment:
prost-build accepts `>=0.4, <=0.5` while `strum_macros` requires `^0.5.0`. Although `0.5.0` satisfies both but it would not be the case for `0.5.1`. The prost-build crate shall fix their declaration to `>=0.4, <0.6`.